packages/gallery/src/components/gallery/proGallery/galleryView.js

```diff
@@ -47,6 +47,11 @@ class GalleryView extends GalleryComponent {
           break;
         case 40: //down
           newIdx = findNeighborItem(idx, 'down');
+          if((this.props.totalItemsCount -1) === newIdx && newIdx === this.state.currentIdx){
```

Comment:
Why not  to leave only (newIdx === this.state.currentIdx) in the cond? When newIdx === this.state.currentIdx?

Reply:
Because the requirement that this only happen is that we are on the last item,

Reply:
If we leave such a condition we can leave the gallery even though we are on the other items in the last row

Reply:
Hey, please add a comment that explains this rule after we now understand what it means